src/Adapter/Currency/CommandHandler/AddCurrencyHandler.php

```diff
@@ -88,4 +93,23 @@ private function assertCurrencyWithIsoCodeDoesNotExist($isoCode)
             );
         }
     }
+
+    /**
+     * @param string $isoCode
+     *
+     * @throws CurrencyConstraintException
+     */
+    private function refreshLocalizedCurrencyData(Currency $currency)
+    {
+        // Following is required when installing new currency on existing languages:
+        // we want to properly update the symbol in each language
+        $context = Context::getContext();
+        $container = isset($context->controller) ? $context->controller->getContainer() : null;
```

Comment:
Please do not load the container this way 😉 . Inject the required service in the constructor:
```
    /**
     * @param int $localeRepoCLDR
     */
    public function __construct($localeRepoCLDR)
    {
        $this->localeRepoCLDR = $localeRepoCLDR;
    }
```
and configure this service dependency injection accordingly in https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Resources/config/services/adapter/currency.yml#L21:
```
    prestashop.adapter.currency.command_handler.create_currency:
        class: 'PrestaShop\PrestaShop\Adapter\Currency\CommandHandler\AddCurrencyHandler'
        arguments:
          - '@prestashop.core.localization.cldr.locale_repository'
```